pre_tasks/gpu_drivers/Archlinux.yml

```diff
@@ -56,5 +56,5 @@
     state: present
   become: true
   when:
-    - is_nvidia
+    - new_nvidia
```

Comment:
Why is this new_nvidia when it should be is_nvidia

Reply:
Because this task installs the mainstream proprietary drivers, which are meant for *new* Nvidia GPUs and not *all* of them  - Trying to run those on an old GPU would have it tell you that the GPU will be completely blacklisted from the system instead of attempting to probe it.